src/base/emu-i386/kvm.c

```diff
@@ -55,9 +55,8 @@
 #define RETURN_MASK ((SAFE_MASK | 0x28 | X86_EFLAGS_FIXED) & \
                      ~X86_EFLAGS_RF) // 0x244dff
 
-extern char _binary_kvmmon_o_bin_end[];
-extern char _binary_kvmmon_o_bin_size[];
-extern char _binary_kvmmon_o_bin_start[];
+extern char _binary_kvmmon_o_bin_end[] asm("_binary_kvmmon_o_bin_end");
```

Comment:
Would it make sense to have that under
ifdef apple?
We can probably even have a macro, like
```
ifdef apple
EXTERN_ASM(x) extern char x[] asm(#x)
else
EXTERN_ASM(x) extern char x[];
endif
```

Reply:
I actually liked the `asm()` better, since it avoids all that preprocessor logic, and it doesn't make the code any less portable (since we're already relying on various GCC/Clang extensions).